api/HTMLMediaElement.json

```diff
@@ -496,10 +496,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "4"
```

Comment:
Are you sure it wasn't present earlier? See below comment for iOS.

Reply:
I'm positive on this one, yeah -- the Chrome version this feature was first supported in provides additional confirmation (by matching WebKit versions).

Reply:
Given the nature of these APIs I wouldn't trust anything based on what Chrome supported when, this was all behind an ifdef in WebKit back then, and as we've seen there were differences on release branches compared to what later showed up on WebKit trunk.

But, since you did test Safari 3.2 and 4 for this PR, I'm confident in the results. It makes some sense, `canPlayType` wasn't really necessary in a first version, it's there to detect differing codec support between browsers. (I don't know exactly how it came about, it was before my time.)